frontend/app/components/modules/project/components/community/fragments/card-content.vue

```diff
@@ -0,0 +1,73 @@
+<!--
+Copyright (c) 2025 The Linux Foundation and each contributor.
+SPDX-License-Identifier: MIT
+-->
+<template>
+  <div class="md:px-6 px-4 py-3">
+    <slot>
+      <div class="flex lg:flex-row flex-col flex-wrap items-start lg:gap-8 gap-4">
+        <div class="flex gap-3 items-stretch flex-1 min-w-0">
+          <!-- Sentiment Icon -->
+          <lfx-community-sentiment-icon :sentiment-label="mention.sentimentLabel" />
+
+          <!-- Text Content -->
+          <div class="flex-1 min-w-0 overflow-hidden">
+            <p
+              v-if="mention.title"
+              class="text-base font-semibold leading-6 text-black mb-3 whitespace-pre-wrap"
+            >
+              {{ mention.title }}
+            </p>
+            <p
+              class="text-base leading-6 text-black whitespace-pre-wrap lg:break-normal break-all md:block hidden"
+              v-html="truncateText(mention.body, 300)"
+            />
+            <p
+              class="text-base leading-6 text-black whitespace-pre-wrap md:hidden block"
+              v-html="truncateText(mention.body, 150)"
```

Comment:
We have function for sanitizing html 
Example: 
v-html="sanitize(config.description(project))"